File: 2654-count-the-number-of-vowel-strings-in-range/count-the-number-of-vowel-strings-in-range.java
class Solution {
    public int vowelStrings(String[] words, int left, int right) {
        int count=0;
        for(int i=left;i<=right;i++){
            char s = words[i].charAt(0);
            char s1=words[i].charAt(words[i].length()-1);
            if(s == 'a' || s == 'e' || s == 'i'|| s == 'o' || s == 'u'){
                if(s1 == 'a' || s1 == 'e' || s1 == 'i'|| s1 == 'o' || s1 == 'u'){
                    count++;
                }
            }
        }
        return count;   
    }
}

File: 2654-count-the-number-of-vowel-strings-in-range/count-the-number-of-vowel-strings-in-range.py
class Solution(object):
    def vowelStrings(self, words, left, right):
        stack=['a','e','i','o','u']
        count=0
        for i in range(left,right+1):
            if words[i][0] in stack and words[i][-1] in stack:
                count+=1
        return count